games-strategy/1oom/1oom-1.0.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="Open source clone of Master of Orion"
+DESCRIPTION="
+1oom is a game engine recreation of the original \"Master of Orion\" (1993), \
```

Comment:
Start the description on the same line as DESCRIPTION, not on a new line

Reply:
Check: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#description

Reply:
Should be resolved in most recent commit, thanks.